includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -230,10 +231,82 @@ private static function _has_active_subscription( $product_id ) {
 	}
 
 	/**
+	 * Get the number of products that have updates.
+	 *
+	 * @return int The number of products with updates.
+	 */
+	public static function get_updates_count() {
+		$cache_key = '_woocommerce_helper_updates_count';
+		if ( false !== ( $count = get_transient( $cache_key ) ) ) {
+			return $count;
+		}
+
+		// Don't fetch any new data since this function in high-frequency.
+		if ( ! get_transient( '_woocommerce_helper_subscriptions' ) ) {
+			return 0;
+		}
+
+		if ( ! get_transient( '_woocommerce_helper_updates' ) ) {
+			return 0;
+		}
+
+		$update_data = self::get_update_data();
```

Comment:
Why doesn't this case set the `_woocommerce_helper_updates_count` transient?

Reply:
Good catch!